modelscope/hub/snapshot_download.py

```diff
@@ -224,7 +233,7 @@ def _snapshot_download(
             directory = os.path.abspath(
                 local_dir) if local_dir is not None else os.path.join(
                     system_cache, 'hub', repo_id)
-            print(f'Downloading Model to directory: {directory}')
+            logger.info(f'Downloading Model to directory: {directory}')
```

Comment:
this shall remain print, otherwise it won't appear in command line usage

Reply:
donw